media.go

```diff
@@ -20,4 +20,5 @@ type RTCTrack struct {
 	Codec       *RTCRtpCodec
 	Packets     <-chan *rtp.Packet
 	Samples     chan<- media.RTCSample
+	RawRTP      chan<- *rtp.Packet
```

Comment:
Is it possible to have a `RTCSampleTrack` and `RTCPacketTrack` ?  Each can embed `RTCTrack` that way we get a little more compile time safety

Reply:
I played around with this. In my opinion it adds a lot of complexity to surrounding code without any corresponding benefit. We would need to change `RTCTrack` to an interface in order to be able to pass the two different track types to the same receivers, none of which actually care about where the input is coming from. I understand the concern about potential user confusion, but I think in practice the way this will be used is to get a new track initialized for either Sample or RTP input and then start feeding it very near by to the initialization code. Not sure if you saw it, but the new initializers actually close the inverse input channel, making it impossible to accidentally send the "wrong" type of input. It's the sort of error that would be discovered quickly in development rather than cropping up in weird bugs later.

Reply:
That sounds perfect to me! My only goal was to not have silent errors, so I am super happy with this